stories/snowflake.py

```diff
@@ -13,7 +13,7 @@
 
 How to make a 3D {paper=material} {snowflake=noun}
 
-Three dimensional {paper=material} {snowflakes=previous_noun_plural} look {magnificent=adjective} hanging in a window or on a wall. Fun for kids or {adults=plural_person}, they are easy to make. Some like them for {Christmas=holiday}, but you may like them any time!
+Three dimensional {paper=material} {snowflakes={snowflake=noun}+plural} look {magnificent=adjective} hanging in a window or on a wall. Fun for kids or {adults=plural_person}, they are easy to make. Some like them for {Christmas=holiday}, but you may like them any time!
```

Comment:
I'm curious: it looks intentional but I don't understand.  What is the difference between `pl` funct and `plural` in the template?

Reply:
I made it so that both '{foos={foo=noun}+plural}' and '{foos={foo=noun}+pl}' will call morph.pl() just so that there's some stylistic flexibility in how you can create the madlib templates. Since both versions look about equally good to me, I didn't want to pick one and then have to bother remembering which it was every time I make a new madlib story. For the same reason, I made it so that both '+prog' and '+ing' will call the progressive function morph.prog(), and both '+part' and '+perf' will call morph.part(), since the particle form of a verb is sometimes called the perfective form.

I'm thinking about doing a similar multi-map to the input messages in mssgs.py, so for example both '{foos=plural_noun}' and 'foos=pl_noun' will get you the plural message, so if people prefer the short hand or prefer the long version they can use either freely.